src/shapes/Text/StyledText.ts

```diff
@@ -260,9 +269,10 @@ export abstract class StyledText<
    */
   getCompleteStyleDeclaration(lineIndex: number, charIndex: number) {
     const style = this._getStyleDeclaration(lineIndex, charIndex) || {},
-      styleObject: TextStyleDeclaration = {};
-    for (let i = 0; i < this._styleProperties.length; i++) {
-      const prop = this._styleProperties[i];
+      styleObject: TextStyleDeclaration = {},
+      styleProps = (this.constructor as typeof StyledText)._styleProperties;
+    for (let i = 0; i < styleProps.length; i++) {
+      const prop = styleProps[i];
```

Comment:
can be refactored to use `pickBy`?

Reply:
maybe/probably but i would love to keep this as types only as much as possible.
Also this function in particular, differently from the toObject we modified with pickBy is render code.